app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
This will be great to have an online participant intake form!!

One thought: instead of duplicating all of these form views, maybe you could just reference the existing ones? That way if we change it in the normal form, it would update for the participants too.